lib/please_cli/please_cli/build.py

```diff
@@ -143,5 +143,94 @@ def cmd(project,
         please_cli.utils.check_result(result, output)
 
 
+@click.command()
+@cli_common.click.taskcluster_options
+@click.argument(
+    'project',
+    required=True,
+    type=click.Choice(please_cli.config.PROJECTS),
+    )
+@click.option(
+    '--nix-build',
+    required=True,
+    default=please_cli.config.NIX_BIN_DIR + 'nix-build',
+    help='`nix-build` command',
+    )
+@click.option(
+    '--docker',
+    required=True,
+    default='/usr/bin/docker',
```

Comment:
Maybe it's better to use `docker` here, so that the command works also if the user has docker in its PATH, even if it's not in /usr/bin/.